submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                </div>
+              </div>
+              <div class="payment-method-choose">
+                <div class="payholder">
+                  <label for="card">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="card"
+                      class="input"
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/visa.svg" alt="" class="img-card" /><img
+                        src="./img/mastercard.svg"
+                        alt=""
+                        class="img-card"
+                      />
+                    </div>
+                  </label>
+                  <label for="paypal">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="paypal"
+                      class="input"
+                      checked
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/paypal.svg" alt="" class="img-pay" />
+                    </div>
+                  </label>
+                  <label for="amazon">
+                    <input
+                      type="radio"
+                      name="paymethod"
+                      id="amazon"
+                      class="input"
+                    />
+                    <div class="select-pay pay-before pay">
+                      <span class="checkedPay"></span
+                      ><img src="./img/amazon.svg" alt="" class="img-pay" />
+                    </div>
+                  </label>
+                </div>
+              </div>
+              <div class="field-text">
+                <div class="field-container field-card">
+                  <h5 class="above-text">CARD NUMBER</h5>
+                  <input
+                    type="text"
+                    inputmode="numeric"
+                    id="card"
+                    class="text text-card"
+                    name="card"
+                    pattern="\d*"
+                    title="Enter your credit card number. Example: 1234 5678 9012 4567"
+                    minlength="16"
+                    maxlength="16"
+                    required
+                  /><span class="cc-number-img"></span>
+                </div>
+                <div class="field-container field-date">
+                  <h5 class="above-text">EXPIRY DATE</h5>
+                  <input
+                    type="text"
+                    inputmode="numeric"
+                    id="card"
+                    placeholder="MM/YY"
+                    class="text text-card"
+                    name="card"
+                    pattern="\d*"
+                    title="Example: 12/22"
```

Comment:
Your title contains 5 symbols: month and year separated by `/`, but you set min length is 4 symbols It's confusing. 
Fix your example text or validation condition.